src/main/java/view_user_public_profile_usecase/ViewUserPublicProfileDSRequestModel.java

```diff
@@ -0,0 +1,14 @@
+package view_user_public_profile_usecase;
+
+
```

Comment:
What is the point of this ds request model? Does it not already do what the ViewPublicProfileRequestModel does? I am a little bit confused.